test/test_helper.bash

```diff
@@ -3,7 +3,7 @@ export TMP="$BATS_TEST_DIRNAME/tmp"
 if [ "$FIXTURE_ROOT" != "$BATS_TEST_DIRNAME/fixtures" ]; then
   export FIXTURE_ROOT="$BATS_TEST_DIRNAME/fixtures"
   export INSTALL_ROOT="$TMP/install"
-  PATH=/usr/bin:/usr/sbin:/bin/:/sbin
+  PATH="/usr/bin:/usr/sbin:/bin/:/sbin:$PATH"
```

Comment:
`/usr/local/bin` is the most important.

I'm guessing some people might compile and install bash
to other directory like `/opt/pkg/bin`.
FreeBSD ports tree supports $PREFIX definition.

c.f. https://www.freebsd.org/doc/en/books/porters-handbook/porting-prefix.html

I don't use this option,
and if you don't want to include whole $PATH,
just adding `/usr/local/bin` satisfies me.